src/pages/using-spark/guides/validation-messaging.mdx

```diff
@@ -0,0 +1,106 @@
+---
+  title: Validation Messaging
+---
+import validationAnatomy from '../../../images/guides/validation-anatomy.png';
+import validationAnatomy2x from '../../../images/guides/validation-anatomy@2x.png';
+export const validationAnatomySet = `${validationAnatomy} 560w, ${validationAnatomy2x} 1120w`
+
+import {
+  SprkHeading,
+} from '@sparkdesignsystem/spark-react';
+
+<span id="validation-messaging-guide" class="docs-b-Link--anchor"></span>
+<SprkHeading element="h1" additionalClasses="sprk-u-mbl sprk-u-Measure" isPageTitle>
+  Validation Messaging Guide
+</SprkHeading>
+
+<span id="main-takeaways" class="docs-b-Link--anchor"></span>
+<SprkHeading element="h2" additionalClasses="sprk-u-mbm sprk-u-Measure" variant="displayThree">
+  Main Takeaways
+</SprkHeading>
+
+Giving validation feedback is an emotional process. Support users with this process by following these guidelines:
+
+- Explain what the Input expects before the user enters data. (Example: password rules, ranges, etc)
+- Give clear and helpful feedback when encountering a problem.
+- Use a combination of [placement](#placement), [timing](#timing), and [language](#language) to guide users through moments of uncertainty.
+
+<img
+  srcSet={validationAnatomySet}
+  class="sprk-u-Measure sprk-u-pam sprk-u-pbh"
+  src={validationAnatomy}
+  alt="Anatomy of input validation with numbers pointing to each part: placement, timing, and language."
+/>
+
+
+<span id="placement" class="docs-b-Link--anchor"></span>
+<SprkHeading element="h2" additionalClasses="sprk-u-mbm sprk-u-Measure" variant="displayThree">
+  Placement
+</SprkHeading>
+
+- Place messaging close to the input. Spark places validation directly under the input.
+- Provide a clear visual to indicate something has gone wrong with user input and needs attention.
+- Plan error messaging for every field.
+- On Forms with multiple fields, show validation messaging for each input field to guide users to a successful submission.
```

Comment:
Capitalization of 'Forms'. Same question as above, it's not capitalized on line 52.

Reply:
good point! Since "forms" aren't a component, I'll make it lower case